test/gtest/uct/cuda/test_cuda_ipc_md.cc

```diff
@@ -41,6 +41,60 @@ class test_cuda_ipc_md : public test_md {
         EXPECT_EQ(CUDA_SUCCESS, cuMemFree(ptr));
         return rkey;
     }
+
+#if HAVE_CUDA_FABRIC
+    static uct_cuda_ipc_rkey_t unpack_masync(uct_md_h md, int64_t uuid)
+    {
+        size_t size = (1 << 22);
```

Comment:
You can use `UCS_MBYTE` etc.
minor: maybe for unit testing purposes it's enough to allocate few KBs?

Reply:
I can go lower but internally all allocations between 1byte and 4MB take about the same time. I think it's ok to use 4MB here.